src/core/ext/transport/chttp2/transport/hpack_encoder.cc

```diff
@@ -171,10 +171,12 @@ static void add_header_data(framer_state* st, grpc_slice slice) {
   }
 }
 
+extern uint8_t* grpc_slice_buffer_tiny_add_non_empty(grpc_slice_buffer* sb,
```

Comment:
any reason why we aren't including a header?

Reply:
That was misplaced, removing.